e2e/setup/linode-secret.yaml

```diff
@@ -0,0 +1,10 @@
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: linode
+  namespace: kube-system
+stringData:
+  token: ${LINODE_TOKEN}
+  region: us-sea
```

Comment:
I wonder if we should set the region to ${LINODE_REGION}. I did run tests with the existing branch with my LINODE_REGION set to us-ord and everything worked fine. I wonder if this region is even used somewhere.

Reply:
Do you mean the region in the secret?

Reply:
yeah